articles/libraries/index.md

```diff
@@ -53,20 +41,13 @@ The Lock widget for each platform has detailed reference documentation.
   </li>
 </ul>
 
-### Lock Support Table
-
-Below are the GitHub links and support status for the various Lock widgets.
+## Lock links and support status
```

Comment:
This should be an H3. For this, and the corresponding heading for SDKs, I think I'd say `Lock repositories and support status` or something rather than the ambiguous "links"?

Reply:
Oops. I think you're correct about the H3. And I'll change the title.